cli/app/app.go

```diff
@@ -1260,6 +1264,8 @@ func Up(c *cli.Context) {
 		dcCreated, err := client.Deployments(api.NamespaceDefault).Create(dc)
 		if err != nil {
 			logrus.Fatalf("Failed to create deployment controller %s: ", k, err)
+		} else {
+			fmt.Println("Deployment controller " + k + " has been created.")
```

Comment:
I'm thinking that we can tell the users the next step, for example `kubectl get deployment, svc` to see what's created (or `kompose ps` after we implemented it)?